pkg/clients/s3/fake/fake.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/iam"
 
 	"github.com/crossplane/provider-aws/apis/storage/v1alpha3"
+
```

Comment:
I don't think this space has to be here, but it appears that CI didn't complain about it so maybe it does?